types/mocha/index.d.ts

```diff
@@ -1142,6 +1142,7 @@ declare namespace Mocha {
 
         constructor(title: string, fn?: Func | AsyncFunc);
 
+        uuid: string;
```

Comment:
Can you point to the relevant part of the original library that shows this property indeed exists on this class? I looked at both the library source code (https://unpkg.com/browse/mocha@10.1.0/lib/runner.js) and documentation (https://mochajs.org/api/runnable), and couldn't find this property.

Reply:
This UUID property is observable in PlainTest interface (Mochawesome) but the currentTest property in Context class is only assignable with Test class which extends Runnable class (Mocha):

![mochawesome](https://user-images.githubusercontent.com/106982785/203532880-c815fd8c-3b9f-4500-9278-04b975c41b1c.png)

![mocha](https://user-images.githubusercontent.com/106982785/203532884-26f2a1fe-4e4c-47b4-a0a2-82dd7ffd2926.png)

Reply:
Here is the type error:

![uuid](https://user-images.githubusercontent.com/106982785/203537092-8b7c08a1-3456-4280-89ae-d6ce3381eade.png)

Reply:
It works with the following inheritance but I would like directly to use the @types/mocha module:

```
/* Extend contexts */
class Identifier extends Test {
  uuid!: string;
  currentTest!: Context;
}
```

Reply:
I understand, but the Mocha library's original source code doesn't seem to have `uuid` as a property of `Runnable` or of `Test`, so it would be incorrect to add this property to these typings. You'd have to make a PR to the mocha library itself to make this property available there first. Do you have any questions? If not, let me know if I can close this PR. Thank you for your understanding.

Reply:
I found Runnable constructor function in "node_modules/mocha/lib/runnable.js" containing ID property key:

![runnable](https://user-images.githubusercontent.com/106982785/203770441-c902af7d-ec28-41d0-86c0-24f225120a75.png)

I am changing the identifier's name now in a new commit...

Reply:
I see, I was looking for a property called `uuid`, but the property on `Runnable` is called `id`, no? According to the line that says `Object.defineProperty(this, 'id', ...)`. Could you update the name of the new property to `id` to reflect that?

Also, from the library's source code, `MOCHA_ID_PROP_NAME` = "__mocha_id__" (https://github.com/mochajs/mocha/blob/41567df1fa039875a043b54b0d4e26153b802893/lib/utils.js#L17)